src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,72 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import React, { useState } from 'react';
+
+type Tab = {
+  id: string,
+  title: string,
+  content: string,
+};
+
+type Props = {
+  tabs: Tab[]
+};
+
+export const Tabs: React.FC<Props> = ({ tabs }) => {
+  const [selectedTab, setToggle] = useState(tabs[0]);
+  const selectedTabId: Tab = tabs.find(
```

Comment:
Replace the `selectedTab` with the `selectedTabId` prop to determine the selected tab.